vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -45,108 +53,139 @@ module IntegersAndNumbers =
     let result1 = sampleFunction1 4573
+
+    // This line uses '%d' to print the result as an integer. This is type-safe.
+    // If 'result1' were not of type 'int', then the line would fail to compile.
     printfn "The result of squaring the integer 4573 and adding 3 is %d" result1
 
-    /// When needed, annotate the type of a parameter name using '(argument:type)'
+    /// When needed, annotate the type of a parameter name using '(argument:type)'.  Parenthesis are required.
     let sampleFunction2 (x:int) = 2*x*x - x/5 + 3
 
     let result2 = sampleFunction2 (7 + 4)
     printfn "The result of applying the 1st sample function to (7 + 4) is %d" result2
 
     /// Conditionals use if/then/elid/elif/else.
     ///
-    /// Note that F# uses whitespace indentation-aware syntax like Python.
+    /// Note that F# uses whitespace indentation-aware syntax, similar to languages like Python or Ruby.
     let sampleFunction3 x = 
         if x < 100.0 then 
             2.0*x*x - x/5.0 + 3.0
         else 
             2.0*x*x + x/5.0 - 37.0
 
     let result3 = sampleFunction3 (6.5 + 4.5)
-    printfn "The result of applying the 2nd sample function to (6.5 + 4.5) is %f" result3
 
+    // This line uses '%f' to print the result as a float.  As with '%d' above, this is type-safe.
+    printfn "The result of applying the 2nd sample function to (6.5 + 4.5) is %f" result3
 
 
-/// Strings and booleans are fundamental data types in F# programming.
-module StringManipulation = 
+/// Booleans are fundamental data types in F#.  Here are some examples of Booleans and conditional logic.
+///
+/// To learn more, see: 
+///     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/primitive-types
+///     and
+///     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/symbol-and-operator-reference/boolean-operators
+module Booleans =
 
-    /// Booleans use 'true', 'false', 'not', '&&' and '||'
+    /// Booleans values are 'true' and 'false'.
     let boolean1 = true
     let boolean2 = false
 
+    /// Operators on booleans are 'not', '&&' and '||'.
     let boolean3 = not boolean1 && (boolean2 || false)
 
-    printfn "The expression 'not boolean1 && (boolean2 || false)' is %A" boolean3
+    // This line uses '%b'to print a boolean value.  This is type-safe.
+    printfn "The expression 'not boolean1 && (boolean2 || false)' is %b" boolean3
+
+
+/// Strings are fundamental data types in F#.  Here are some examples of Strings and basic String manipulation.
+///
+/// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/strings
+module StringManipulation = 
 
     /// Strings use double quotes.
     let string1 = "Hello"
     let string2  = "world"
 
-    /// Strings can also use @ to create a verbatim string literal
-    let string3 = @"c:\Program Files\"
+    /// Strings can also use @ to create a verbatim string literal.
```

Comment:
Maybe explain that verbatim strings will ignore escape sequences such as \n, \t etc. It may also be useful to demonstrate the logical equivalent: "C:\\Program Files\\"